runtime/convertValues.go

```diff
@@ -488,7 +488,9 @@ func importCompositeValue(
 		case sema.SignatureAlgorithmType, sema.HashAlgorithmType:
 			// continue in the normal path
 		default:
-			panic(fmt.Errorf("cannot import value of type %s", qualifiedIdentifier))
+			// Continue and return a dummy value.
+			// It will be validated later through the 'IsStorable' dynamic check.
+			// TODO: return an error
```

Comment:
I have another PR that does refactor this to return an error, for now, can we just keep the panic and not silently accept everything?